### PR TITLE
feat(icon): DLT-1804 add t-mobile

### PR DESCRIPTION
**Pull request by fede-dp**
# DLT-1804 add t-mobile

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [x] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [ ] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket

DLT-1834(https://dialpad.atlassian.net/browse/DLT-1834)

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

## :camera: Screenshots / GIFs

image

### CODE REVIEW

File: packages/dialtone-icons/src/icons.json
Comment (francisrupert) [inline packages/dialtone-icons/src/icons.json:1]: I'm unclear on this file. Obviously it's autogenerated, but it seems to be new... for what reason? Who can answer that @juliodialpad perhaps?

File: packages/dialtone-icons/src/keywords-icons.json [hunk 1]
```diff
@@ -272,6 +272,9 @@
       "slack": [],
       "snapchat": [],
       "telegram": [],
+      "t-mobile": [
+        "tmo"
```
Comment (francisrupert) [inline packages/dialtone-icons/src/keywords-icons.json:276]: Perfect keyword.

**Review by francisrupert — APPROVED** [review]
I approve from Design perspective. Before merging, resolve my question about `icons.json` from a DT engineer.

File: packages/dialtone-icons/src/icons.json
Comment (iropolo) [reply packages/dialtone-icons/src/icons.json:1]: That file should not be included in this PR, not sure why it is there but it is not in our main branch.
It is not part of the workflow of icon addition.

**Review by iropolo — CHANGES_REQUESTED** [review]
Remove `icons.json`.

The rest looks fine.

File: packages/dialtone-icons/src/icons.json
Comment (braddialpad) [reply packages/dialtone-icons/src/icons.json:1]: @fede-dp how is this file even showing up? Is it getting generated when you build or run some other process?

**Comment by fede-dp** [comment]
> Remove `icons.json`.
> 
> The rest looks fine.



> Remove `icons.json`.
> 
> The rest looks fine.



> Remove `icons.json`.
> 
> The rest looks fine.

removed!

**Review by braddialpad — APPROVED** [review]
Looks fine now, any idea what you did to make that file generate?

**Review by iropolo — APPROVED** [review]

**Pull request merged**